snprc_ehr/resources/queries/study/BrowseDataSets.sql

```diff
@@ -68,3 +68,5 @@ UNION
 select 'snprc_ehr' as schema, 'ClinPath' as CategoryId,'Urinalysis Results' as Label, 'HL7UrinalysisPivot' as Name,  true as ShowByDefault, true as isAnimal
 UNION
 select 'ehr' as schema, 'Colony Management' as CategoryId,'Kinship' as Label, 'kinship' as Name,  true as ShowByDefault, true as isAnimal
+UNION
+select 'ehr' as schema, 'Colony Management' as CategoryId,'Project' as Label, 'project' as Name,  true as ShowByDefault, false as isAnimal
```

Comment:
Is this change on purpose Terry? 'Project' as Label, 'project' as Name, instead of 'Kinship' as Label, 'kinship' as Name,

Reply:
the table name is ehr.project and the label for it is project

Reply:
This wasn't a change. It was an additional table